tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -413,38 +413,6 @@ public class TLAplusParser implements tla2sany.st.SyntaxTreeConstants, ParseTree
     currentT.next = nextT;
   }
 
-//
-  void skipOver( int l ) {
-    while ( true ) {
-      Token t = getToken(1);
-      int k = t.kind;
-      if ( (k == EOF) || (t.beginColumn < l ) ) return;
-      t = getNextToken();
-    }
-  }
-
-/***************************************************************************
-* Note: the non-terminal ClosedStart was commented out, apparently to be   *
-* replaced by this boolean-valued method.                                  *
-***************************************************************************/
-boolean
-ClosedStart( Token t ) {
```

Comment:
Farther down there is a comment that mentions this method:
```
// op_119 : dernier compteur : impact sur ClosedStart et autres !!!
```

The comment is in French. It should be updated, and if we're doing that we might as well also translate it to English, something like:

```
// NOTE: op_119 is the last operator token, and it is used in helper methods to detect certain classes of tokens.
// We have to be careful to adjust those uses if we ever add new operator tokens.
```